src/main/target/common_pre.h

```diff
@@ -333,4 +333,5 @@
 #define USE_VTX_TABLE
 #define USE_PERSISTENT_STATS
 #define USE_PROFILE_NAMES
+#define USE_SERIALRX_SRXL2     // Spektrum SRXL2 protocol
```

Comment:
There's a problem with code missing proper bounding with `USE_SERIALRX_SRXL2`.

Disabling the `#define USE_SERIALRX_SRXL2` will result in compilation errors. Likely you have some unbounded references to code that is only included when `USE_SERIALRX_SRXL2` is defined.

```
/var/folders/75/1m9wjhn08xj5g10059b9s2sr0000gn/T//ccquXECb.ltrans2.ltrans.o: In function `taskTelemetry.lto_priv.420':
<artificial>:(.text+0x469e): undefined reference to `srxl2TelemetryRequested'
/var/folders/75/1m9wjhn08xj5g10059b9s2sr0000gn/T//ccquXECb.ltrans6.ltrans.o: In function `processSrxl.lto_priv.1131':
<artificial>:(.text+0x5ae2): undefined reference to `srxl2InitializeFrame'
<artificial>:(.text+0x5b5a): undefined reference to `srxl2FinalizeFrame'
/var/folders/75/1m9wjhn08xj5g10059b9s2sr0000gn/T//ccquXECb.ltrans6.ltrans.o: In function `telemetryInit':
<artificial>:(.text+0x64b2): undefined reference to `srxl2RxIsActive'
```

Reply:
@etracer65 Is it ok if I do a force push for this fix or would it be better for the initial commit to be separate, and then squash?

Reply:
Went ahead and just made a separate commit for now. Let me know if I can squash

Reply:
Looks good. You can go ahead and squash.

Reply:
Done and pushed.